### ISSUE TITLE
adding support for setting node path

### ISSUE THREAD

**Issue by oriSomething** [felixrieseberg/vsc-ember-cli#16]
i have a problem that i need `nvm` for multiple versions of `node`, and since the default version is too old for using of `ember-cli`, i need a way to hint the extension where is `ember-cli`

**Comment by felixrieseberg** [comment]
Good call, that makes a lot of sense. We already have a little settings manager - how would you feel about tacking a crack at that issue? I'd :heart: to see a PR!

**Comment by oriSomething** [comment]
i'm not sure... i start to thing it's actually a VSCode issue rather than this extension

**Comment by oriSomething** [comment]
i've opened [this issue](https://github.com/Microsoft/vscode/issues/5456)

**Comment by ctsstc** [comment]
Can we close this one out @felixrieseberg ?
